Reverted, removed the second hunk about dev-libs/check, and further
adjusted. The tarball already contains the man page and gengetopt
generated command line parsing, so these checks are useless to us.

From fbd340d1c777e23b66ef27ce1c03c4eb3bafc511 Mon Sep 17 00:00:00 2001
From: Thordur Bjornsson <thorduri@yubico.com>
Date: Thu, 12 Jan 2017 14:28:31 +0100
Subject: [PATCH] auto: Check for help2man/gengetopt, and bail out if not
 found.

Both are required to build.

While here, if `--enable-tests` passed, check for `check` and
if not found, bail out as well.
---
 configure.ac | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index ad409a2..6decb78 100644
--- b/configure.ac
+++ a/configure.ac
@@ -43,14 +43,7 @@
 LT_INIT([win32-dll])
 
 AM_MISSING_PROG(HELP2ADOC, help2adoc, $missing_dir)
 AM_MISSING_PROG(HELP2MAN, help2man, $missing_dir)
-if ! which help2man >/dev/null ; then
-   AC_MSG_ERROR([help2man missing])
-fi
-if ! which gengetopt >/dev/null ; then
-   AC_MSG_ERROR([gengetopt missing])
-fi
-
 gl_LD_VERSION_SCRIPT
 
 PKG_CHECK_MODULES([LIBJSON], [json-c], [], [
